stardog_loader.py

```diff
@@ -0,0 +1,30 @@
+import fire
+import json
+import os
+
+def handle_data(*dirs):
+    for d in dirs:
+        for kubecontext, ns, pod in process(d, "pods"):
+            print(kubecontext, ns, pod["metadata"]["name"])
+
+        for kubecontext, ns, namespace in process(d, "namespaces"):
+            print(kubecontext, ns, namespace["metadata"]["name"])
+
+        for kubecontext, ns, deployment in process(d, "deployments"):
+            print(kubecontext, ns, deployment["metadata"]["name"])
+
+
+def process(thedir, kind):
+    kubecontext = os.path.basename(os.path.normpath(thedir))
+    print(kubecontext)
+    filename = os.path.join(thedir, "{}.json".format(kind))
+    print(filename)
+    with open(filename) as json_file:
+        items_wrapper = json.load(json_file)
+        for item in items_wrapper["items"]:
```

Comment:
`items` is guaranteed to be present, yes? :+1:

Reply:
yes - k8s is great about this and you'll get an empty array if there was none of something...

```
$ kubectl get daemonset -o json | jq
{
  "apiVersion": "v1",
  "items": [],
  "kind": "List",
  "metadata": {
    "resourceVersion": "",
    "selfLink": ""
  }
}
```